api/app/src/external/commonwell/api.ts

```diff
@@ -78,33 +77,38 @@ export const metriportQueryMeta: RequestMetadata = baseQueryMeta("Metriport");
 
 const commonwellCertificate = Config.getMetriportCert();
 const commonwellCertificateContent = getCertificateContent(commonwellCertificate);
-const x509 = new X509Certificate(commonwellCertificate);
-
-const thumbprint = x509.fingerprint;
 
-const validFrom = dayjs(x509.validFrom).toString();
-const validTo = dayjs(x509.validTo).toString();
+export function getCertData() {
```

Comment:
Turned to functions so we can mock those - otherwise would need valid certs on tests